lib/capsize/deployer.rb

```diff
@@ -196,7 +201,8 @@ def handle_error(error) #:nodoc:
       when Net::SSH::AuthenticationFailed
         logger.important "authentication failed for `#{error.message}'"
       else
-        logger.important error.message + "\n" + error.backtrace.join("\n")
+        @deployment.log = (@deployment.log || '') + error.message + "\n" + error.backtrace.join("\n")
+        @deployment.save!
```

Comment:
Please create an `append_log` method for `@deployment`, and use it both here and up on line 112. You've got extremely similar code in two places, and it'll make life easier for reactoring.